src/devices/lumi.ts

```diff
@@ -2916,21 +2916,21 @@ const definitions: Definition[] = [
         ],
         meta: {multiEndpoint: true},
         endpoint: (device) => {
-            return {'button': 1};
+            return {'button_1': 1};
```

Comment:
Since this is a single rocker, why is multi endpoint needed in the first place? Can you try without `meta` and `endpoint: (device)`?